angelsbioprocessing/data.lua

```diff
@@ -54,7 +54,9 @@ end
 angelsmods.triggers.bio_rubber = angelsmods.triggers.bio_rubber or angelsmods.trigger.rubber or false
 angelsmods.triggers.bio_resin = angelsmods.triggers.bio_resin or angelsmods.trigger.resin or false
 angelsmods.triggers.bio_plastic = angelsmods.triggers.bio_plastic or (angelsmods.trigger.plastic and (angelsmods.triggers.bio_resin or angelsmods.triggers.bio_rubber)) or false
+
 angelsmods.triggers.paper = not angelsmods.functions.is_special_vanilla()
+angelsmods.triggers.crystals_full = (not angelsmods.functions.is_special_vanilla()) or (mods["bobmodules"] and true or false)
```

Comment:
@KiwiHawk how comfortable are you with this trigger mechanism, it looks awkward.
do you want it to be true with bobmodules without plates? (this would trigger with just angels and bobmodules without plates?)
i guess... what I'm asking is, does the trigger work as intended?

Reply:
Yes, I'm comfortable with triggers. And yes, this is working as intended. Crystals are added to the recipes for Bob's modules, even in special vanilla mode.